test/node/mock-generator.js

```diff
@@ -3,7 +3,7 @@
 require('yeoman-generator');
 var _ = require('lodash');
 
-module.exports = function() {
+module.exports = function() {
```

Comment:
I suppose it was non breakable space. Mac unfortunatly allow to create them really to easily (alt-space)